src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -595,22 +604,15 @@ function make_compute_exp_tendency(
         @inbounds for i in 1:(n_stem + n_leaf)
             im1 = i - 1
             ip1 = i + 1
-            # To prevent dividing by zero, change AI/(AI x dz)" to
-            # "AI/max(AI x dz, eps(FT))"
-            AIdz =
-                max.(
-                    getproperty(area_index, model.compartment_labels[i]) .* (
-                        model.compartment_surfaces[ip1] -
-                        model.compartment_surfaces[i]
-                    ),
-                    eps(FT),
-                )
+            # To prevent dividing by zero, change AI/(AI x dz)" to AI/max(AI x dz, eps(FT))"
```

Comment:
These changes were for readability

Reply:
this area index should be the "center" area index, not the face, so no interp c 2 f needed